Tests/Microsoft.Teams.Common.Tests/Microsoft.Teams.Common.Tests.csproj

```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Libraries\Microsoft.Teams.Api\Microsoft.Teams.Api.csproj" />
     <ProjectReference Include="..\..\Libraries\Microsoft.Teams.Common\Microsoft.Teams.Common.csproj" />
+    <ProjectReference Include="..\..\Libraries\Microsoft.Teams.Plugins\Microsoft.Teams.Plugins.AspNetCore.DevTools\Microsoft.Teams.Plugins.AspNetCore.DevTools.csproj" />
```

Comment:
tests for plugins should be in their on test project, ie `Microsoft.Team.Plugins.AspNetCore.Devtools.Tests`